Jenkinsfile

```diff
@@ -1,7 +1,9 @@
 pipeline {
 
     agent {
-        label 'camtango_db'
+        // TODO: Update with the appropriate docker image once PR(https://github.com/ska-sa/kattrap/pull/90)
```

Comment:
I see that the mentioned PR has been merged. Do we still need to keep this `TODO`?